.github/workflows/bootstrap_archives.yml

```diff
@@ -74,7 +75,7 @@ jobs:
         uses: termux/upload-release-action@v3.0.3
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: "*.zip"
+          file: "*.zip *.sha256sum"
```

Comment:
You can't pass multiple files or globs.

https://github.com/svenstaro/upload-release-action#input-variables

https://github.com/svenstaro/upload-release-action/issues/34

https://github.com/termux/termux-packages/actions/runs/3070386375

Reply:
Please check my PR (#11960) and see is it correct (I am quite a newbie here).